src/main/java/com/google/devtools/build/lib/runtime/StarlarkOptionsParser.java

```diff
@@ -228,7 +230,21 @@ private Target loadBuildSetting(
     ImmutableList.Builder<String> keep = ImmutableList.builder();
     ImmutableList.Builder<String> remove = ImmutableList.builder();
     for (String name : list) {
-      ((name.startsWith("--//") || name.startsWith("--no//")) ? remove : keep).add(name);
+      if (!name.startsWith("--")) {
```

Comment:
This logic is great, but needs comments. More comments.

Reply:
Done! added a description to the javadoc as well